lib/algo/greedy/GreedyRandomSampler.cpp

```diff
@@ -145,7 +145,7 @@ GreedyRandomSampler::constructParameterPoints()
     if (d_use_latin_hypercube)
     {
         std::vector<double> frequencies;
-        std::vector<std::vector<double>> point_coordinates;
+        std::vector<std::vector<double> > point_coordinates;
```

Comment:
SWIG does not allow >>, it requires a space in between, so I updated these to > >